providers/manage.rb

```diff
@@ -83,6 +83,7 @@ def cert_file_resource(path, content, options = {})
     mode(options[:private] ? 00640 : 00644)
     variables :file_content => content
     only_if { content }
+    sensitive new_resource.sensitive
```

Comment:
As this only came in a very recent Chef release, and we don't want to break backwards compatibility, you will most probably need to check if that attribute exists. I think this should work:
```ruby
sensitive(true) if respond_to?(:sensitive)
```